### PR TITLE
Update development-builds.md

### PR DESCRIPTION
**Pull request by AshtakaOOf**
Add copr nightly
Other PR https://github.com/PrismLauncher/PrismLauncher/pull/407

Signed-off-by: Adrien <66513643+AshtakaOOf@users.noreply.github.com>

### CODE REVIEW

**Comment by AshtakaOOf** [comment]
fuck you lint

**Comment by AshtakaOOf** [comment]
Forgor to sign off :skull: I hate my life goodnight

**Comment by getchoo** [comment]
ditto https://github.com/PrismLauncher/PrismLauncher/pull/407#discussion_r1014769980

**Comment by AshtakaOOf** [comment]
Yes

File: src/wiki/development/development-builds.md [hunk 1]
```diff
@@ -15,6 +15,6 @@ Non-portable binaries are currently available for Windows as a system install, m
 
 Portable dev-builds are also provided; for both Windows and Linux.
 
-For Arch, Debian and Gentoo, respectively, you can use these packages for the latest development versions:
+For Arch, Debian, Fedora or Opensuse and Gentoo, respectively, you can use these packages for the latest development versions:
```
Comment (DioEgizio) [inline src/wiki/development/development-builds.md:18]:
```suggestion
For Arch, Debian, Fedora or OpenSUSE and Gentoo, respectively, you can use these packages for the latest development versions:
```

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**